stats/barseries.cpp

```diff
@@ -151,6 +151,7 @@ void BarSeries::BarLabel::updatePosition(bool horizontal, bool center, const QRe
 				setVisible(false);
 				return;
 			}
+			pos.rx() -= itemSize.width() / 2.0;
```

Comment:
elsewhere you point out that things look weird if they end up on non-integer coordinates. 
Is that a concern here as well? Should this be rounded or simply done as integer division?

Reply:
Not a concern - It's rounded two lines down with the `roundPos()` helper function.

Reply:
ah, context matters. not enough context on Github